### DISCUSSION TITLE
Grafana dashboard

### DISCUSSION THREAD

**Discussion by WyriHaximus** [flant/shell-operator#243] [Q&A] [answered]
Couldn't find anything, but is there any Grafana dashboard for this operator somewhere?

**Comment by diafour** [comment, marked as answer]
Hello! There is no dashboard for the shell-operator. Internally we monitor it as a regular Pod and use metrics for Prometheus alerts.

**Comment by WyriHaximus** [comment]
Cool, might have a try at it, just finished my first operator and helm chart based on this operator. Will share here if I have anything
